src/OrchardCore.Themes/TheBlogTheme/Views/Content-Blog.liquid

```diff
@@ -1,7 +1,7 @@
 {% zone "Header" %}
 <!-- Page Header -->
 <!-- Set your background image for this header on the line below. -->
-<header class="masthead" style="background-image: url('{{ "~/TheBlogTheme/img/home-bg.jpg" | href }}')">
+<header class="masthead" style="background-image: url('{{ Model.ContentItem.Content.Blog.Image.Paths[0] | asset_url }}')">
```

Comment:
```suggestion
<header class="masthead" style="background-image: url('{{ Model.ContentItem.Content.Blog.Image.Paths.first | asset_url }}')">
```

Reply:
Can you also an an `{% if %}` to define the actual url to generate, such that if this is not defined ( `== empty`) then it should use `"~/TheBlogTheme/img/home-bg.jpg" | href` as a fallback. This will prevent existing sites from breaking, and allow a blog/blogpost to not define a value in the media field.

Reply:
I don't think we need this because the cover image is required field

Reply:
We need this to guarantee that blogs that have been created before this change will continue to work as previously, when the fields did not exist.
It is always important to think about backward compatibility.

Reply:
I didn't think about this, I will do it now ..